test/saithrift/tests/switch.py

```diff
@@ -85,7 +84,19 @@ def switch_init(client):
     for interface,front in interface_to_front_mapping.iteritems():
         sai_port_id = client.sai_thrift_get_port_id_by_front_port(front);
         port_list[int(interface)]=sai_port_id
-           
+        
+    for num_of_tries in range(20):
+        # wait till the port are up
```

Comment:
we should wait till all port to be up. all the ports that have mapped in the interfaces.